### PR TITLE
Fix empty line detection in markdown in list

### PR DESCRIPTION
**Pull request by karelbilek**
Fix for independent issue in #177

The next parser should have the whitespace removed, even when it's blank

### CODE REVIEW

**Comment by karelbilek** [comment]
Rebased, fixed the test so they actually test the bug

**Comment by karelbilek** [comment]
@yuin planning to merge this? (sorry for pushing :D )

File: parser/list.go [hunk 1]
```diff
@@ -164,6 +164,8 @@ func (b *listParser) Continue(node ast.Node, reader text.Reader, pc Context) Sta
 		if node.ChildCount() == 1 && node.LastChild().ChildCount() == 0 {
 			return Close
 		}
+
+		reader.Advance(len(line)-1)
```
Comment (13rac1) [inline parser/list.go:168]: nit: A comment explaining "why" would be helpful here

**Review by 13rac1 — APPROVED** [review]
Straightforward change. I wish I could merge

File: parser/list.go (hunk 1 above)
Comment (karelbilek) [reply parser/list.go:168]: Hm I already forgot :D it would get me a while to get into this

Comment (13rac1) [reply parser/list.go:168]: 😆 That's why I write the "why" comments

**Comment by karelbilek** [comment]
thanks!!

**Pull request merged**